### ISSUE TITLE
What quadrants make sense for Redgate?

### ISSUE THREAD

**Issue by fffej** [red-gate/Tech-Radar#4]
The original Thoughtworks radar gives four areas
* Techniques
* Tools
* Platforms
* Languages and Frameworks

Are these the right ones for Redgate? What four should we pick?

**Comment by tugberkugurlu** [comment]
All seems sensible and can relate to us apart from "Programming Languages and Frameworks" thing. I might be worth separating these two out into "Web Development Frameworks and Libraries" and ".NET Development Programming Languages and Frameworks" maybe, or something along the lines of this.

The main outcome I expect by this is to be able to easily spot relevant things under each category. For instance, for Web one, I would like to see one Programming Language under "Adopt" and just go with it as a new team who is just starting out with this. Same for .NET one. If I am looking for an IoC container, I should be able to spot only one under "Adopt".

There is a value to be open-mined about other Programming Languages outside of .NET land but it's just not a reality for us I believe. Not sure how fair is assumption though.

**Comment by smobs** [comment]
I like _languages and frameworks_ grouped.  Frameworks are a similar commit, and likely to be less well designed

**Comment by smobs** [comment]
> The main outcome I expect by this is to be able to easily spot relevant things under each category. For instance, for Web one, I would like to see one Programming Language under "Adopt" and just go with it as a new team who is just starting out with this. Same for .NET one. If I am looking for an IoC container, I should be able to spot only one under "Adopt".

I agree with Tugberk and wonder if the radar is just a visualisation of a different source of truth.  
Something more like a spreadsheet of 

|theme| what  | explore |  **adopt** | endure | retire |
|------|-------|--------|--------|---------|------|
|Tools| AOC | hand rolling | **Autofac** | Ninject | RhinoMoc |

**Comment by fffej** [comment]
@smobs I think Platform for us probably means supported SQL Server versions, Windows versions, Web Browsers and .NET framework. 

@smobs I think that observation around the spreadsheet is good (except I imagine you'd need multiple things in endure for that particular example 😄

**Comment by nyctef** [comment]
+1 to what @smobs said - the radar is a cool visualization, but I think the spreadsheet would actually be more useful on a day-to-day basis

**Comment by smobs** [comment]
Multiple things in retire Jeff

**Comment by smobs** [comment]
Also RhinoMoc is not right

**Comment by smobs** [comment]
Bold off topic statement, a IOC systems should be straightfoward to migrate, because you rewrite all the configuration in the new system and don't touch real code.

**Comment by fffej** [comment]
The more I think about this, the more I think we should just adopt the Thoughtworks ones. They aren't _perfect_, but as the [FAQ](https://www.thoughtworks.com/radar/faq) says

> We don't make a big deal out of the quadrants - they’re really just a way to break up the Radar into topic areas. We don't think it's important which quadrant a blip goes into ...

I don't think we should spend much more time discussing this.

@tugberkugurlu makes a good point about easily wanting to spot relevant things, but I think that will come from a different presentation of the same data (or just over time as we start to rationalize the stuff we use 😄).
